externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/SparkSQLEngine.scala

```diff
@@ -160,7 +160,9 @@ object SparkSQLEngine extends Logging {
       "20")
 
     val appName = s"kyuubi_${user}_spark_${Instant.now}"
+    val podNamePrefix = s"kyuubi_${user}_spark_sql"
     _sparkConf.setIfMissing("spark.app.name", appName)
+    _sparkConf.setIfMissing("spark.kubernetes.executor.podNamePrefix", podNamePrefix)
```

Comment:
In kubernetes, same pod name is not supported. This may cause same user's another application can't scala up executor pod. How about add some uuid?